packages/frontend/src/components/ShortcutMenu/index.tsx

```diff
@@ -38,6 +51,16 @@ export default function ShortcutMenu(props: Props) {
         onClick={props.showContextMenu}
         tabIndex={props.tabindexForInteractiveContents}
       />
+      {props.message.originalMsgId !== null && (
+        <button
+          className={classNames(styles.originalMessageButton)}
+          aria-label={tx('jump_to_original_message')}
+          onClick={onClick}
+          title={tx('jump_to_original_message')}
```

Comment:
this is what we're using on android / iOS, it is good enough, and also already known from other places.

we can iterate, if we get corresponding feedback at scale

```suggestion
          aria-label={tx('show_in_chat')}
          onClick={onClick}
          title={tx('show_in_chat')}
```

Reply:
Will change it again once you added the new string...